globus_sdk/config.py

```diff
@@ -154,16 +154,27 @@ def _bool_cast(value):
     raise ValueError("Invalid config bool")
 
 
+def _get_token_type(tok_type, environment):
```

Comment:
Nitpick: I think this should be named _get_token(token_type, environment). I was expecting it to return a token type, not a token of the specified type.